src/planscape/conditions/management/commands/load_conditions.py

```diff
@@ -66,17 +77,22 @@ def process_metric(self, metric):
             region_name=metric["region_name"],
             condition_level=ConditionLevel.METRIC,
         )
+        # FIXME: if we ever start using normalized metrics again we need to change this
```

Comment:
unless SIG uses FIXME as the label for things like this, we have a number of lines in the code that use "TODO:"